astropy_helpers/sphinx/conf.py

```diff
@@ -135,15 +135,11 @@ def check_sphinx_version(expected_version):
     'sphinx.ext.coverage',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.viewcode',
-    'astropy_helpers.sphinx.ext.numpydoc',
-    'astropy_helpers.sphinx.ext.astropyautosummary',
```

Comment:
👍 to finally remove it. I'm not sure whether we need to mention this in the changelog as a removed deprecation.